bin/check-haproxy.rb

```diff
@@ -80,6 +80,14 @@ class CheckHAProxy < Sensu::Plugin::Check::CLI
          default: 90,
          proc: proc(&:to_i),
          description: 'Session Limit Critical Percent, default: 90'
+  option :backend_session_warn_percent,
+         short: '-b PERCENT',
+         proc: proc(&:to_i),
+         description: 'Per Backend Session Limit Warning Percent, default: 75'
```

Comment:
It doesn't use the default Session Limit Warning, the purpose of this was to not get paged when only one service has used up all it's sessions. We have 25 services per backend so it isn't important for us to monitor each and every single service. That's why the code is setup to ignore  Session Limit Critical if Backend Session Limit Critical is set and doesn't do anything if it isn't set